Makefile

```diff
@@ -615,6 +615,7 @@ cleanall: clean
 .PHONY: cleanappcache
 cleanappcache:
 	rm -f prd/geoadmin.appcache
```

Comment:
I want to make sure we don't have this file anywhere. appcache made me paranoid :)